runserver.py

```diff
@@ -10,7 +10,7 @@
 _server = _servers[0]
 _host = _hosts[0]
 _port = 8080
-_ui = False
+_ui = True
```

Comment:
As mentioned above, please don't change default values to avoid changing the production behavior. We should drive local development with settings in `makefile` and/or `docker-compose.yml` which are our developer tools.

Reply:
👍🏽